crates/moonutil/src/lib.rs

```diff
@@ -16,6 +16,8 @@
 //
 // For inquiries, you can contact us via e-mail at jichuruanjian@idea.edu.cn.
 
+#![warn(clippy::clone_on_ref_ptr)]
```

Comment:
can we make this a workspace level config?

Reply:
have no idea how to do this🤣